internal/dataplane/kong_client.go

```diff
@@ -520,10 +532,16 @@ func handleSendToClientResult(client sendconfig.KonnectAwareClient, logger logru
 		}
 		return "", err
 	}
-
 	return newSHA, nil
 }
 
+func (c *KongClient) SetConfigStatusNotifier(n ConfigStatusNotifier) {
```

Comment:
Can we please add a comment here mentioning e.g. who might be the primary consumer of this notification?

Reply:
added.